README.md

```diff
@@ -2,3 +2,163 @@
 
 An initial scalable filesystem for Linux operating systems
 
+- [Why use initoverlayfs to generate initramfs images?](#why-use-initoverlayfs-to-generate-initramfs-images-)
+- [Installation](#installation)
+    * [Step 1 - Deploy the software](#step-1---deploy-the-software)
+    * [Step 2 - Run initoverlayfs-install](#step-2---run-initoverlayfs-install)
+    * [Step 3 - Reboot to test](#step-3---reboot-to-test)
+    * [Step 4 - Validating the boot](#step-4---validating-the-boot)
+
+# Why use initoverlayfs to generate initramfs images?
+
+An initramfs (Initial RAM File System) image is a fundamental component in preparing Linux systems during the boot process, preceding the initiation of the init process. 
+
+Typically, generating an initramfs involves assembling all available kernel modules and necessary files to boot and support any hardware using the specific Linux kernel version XYZ.  
+
+However, this conventional approach presents a significant challenge: loading such a voluminous image into memory during boot is time-consuming and can be problematic in critical scenarios such as healthcare, automotive, and aviation.
```

Comment:
problematic in critical scenarios where time to boot is critical.  Edge devices commonly need to boot as quickly as possible such as healthcare, automotive and aviation.